### PR TITLE
Fixed argument parser so it throws an error when given multiple value…

### PR DESCRIPTION
**Pull request by aamayorga**
…s are provided for a single value argument

### CODE REVIEW

**Comment by hartbit** [comment]
@aciidb0mb3r This is a PR made at try! Swift San Jose. I checked it and sounds good. What do you think?

File: Tests/FunctionalTests/XCTestManifests.swift [hunk 1]
```diff
@@ -27,32 +27,6 @@ extension DependencyResolutionTests {
     ]
 }
 
-extension MiscellaneousTestCase {
```
Comment (aciidb0mb3r) [inline Tests/FunctionalTests/XCTestManifests.swift]: This diff and removal of the `MiscellaneousTests.swift` file seems like a mistake? Can we revert that?

**Comment by aciidb0mb3r** [comment]
Looks good! Can you please revert the unintended diff?

**Comment by hartbit** [comment]
@aamayorga Let me know if you need any help with fixing the diff!

**Comment by aamayorga** [comment]
Sorry about that! I believe that commit reverted the unintended changes.

**Comment by aciidb0mb3r** [comment]
@swift-ci smoke test

**Pull request merged**